app.js

```diff
@@ -108,14 +108,22 @@ var q = new ProcessQueue();
 
 function setupWorkDirectory() {
     try {
-        fs.readdirSync(tempPath).forEach(fname => {
-            fs.unlinkSync(tempPath + '/' + fname);
-        });
-        fs.rmdirSync(tempPath);
-    } catch(e) {
-        console.error('work dir does not exist');
+        if (fs.existsSync(tempPath)) {
+            fs.readdirSync(tempPath).forEach(fname => {
+                try {
+                    console.log('Removing file ' + tempPath + '/' + fname);
```

Comment:
If this is a newer version of node, you can use the es6 feature template literals. Eg `<backtick>Removing file ${tempPath}/${fname}<backtick>`

Reply:
I opted to keep the legacy format here as the rest of the codebase does not use template literals. If we wanted to move to template literals it might be best done in a refactor ticket? Any thoughts?

Reply:
Yeah makes sense. It probably depends on this codebase. If it's something we're maintaining, then usually we refactor as we make changes. But since this is a fork of another project, we may want to be careful not to get too different for the sake of merges from the original.